tcpdump/module.info

```diff
@@ -6,5 +6,5 @@
         "tetra"
     ],
     "title": "tcpdump",
-    "version": "1.5"
-}
\ No newline at end of file
+    "version": "1.5.1"
```

Comment:
Why not 1.6?

Reply:
whoopps n00b here to github PRs. Did a comment instead of a reply. I changed to 1.6 and did new commit. LMK if anything else.